Scripts/run_tests.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -Eeuo pipefail
+
+readonly SCRIPT_ABS_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd -P)"
+
+readonly PROJECT_ROOT_PATH="$SCRIPT_ABS_PATH/.."
+
+readonly PLATFORM_IOS="iOS Simulator,name=iPhone 14 Pro"
+readonly PLATFORM_TVOS="tvOS Simulator,name=Apple TV 4K (3rd generation) (at 1080p)"
+
+readonly XCODE_PROJECT="FingerprintJS.xcodeproj"
+readonly XCODE_SCHEME="FingerprintJS"
+
+cd "$PROJECT_ROOT_PATH"
+
+function xcodebuild_test() {
+    xcodebuild test \
```

Comment:
This command doesn't seem to respect the `xcodebuild` man page synopsis (i.e. the order of arguments might be wrong).

Reply:
I took a look at the man page of `xcodebuild` and I suppose that your concern is that the `test` action should rather be the last parameter according to the *SYNOPSIS* section. But then, it doesn't say anywhere that the order of the parameters must be strictly obeyed. Moreover, if you go all the way down to the *EXAMPLES* section of the man page you will find the following:
<img width="1338" alt="Screenshot 2022-11-29 at 09 40 20" src="https://user-images.githubusercontent.com/10102561/204486426-be57d21b-2de0-48bc-bb66-db8114681ac6.png">
Thus, I suppose that it is okay to leave the current command as is.